.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -29,6 +29,36 @@ assignees: freitagb, wildintellect
 **Intended science use case**
 *Please describe how you intend to use the data, or the expected relevance to MAAP users.*
 
+## Spatio-Temporal Asset Catalog (STAC)
```

Comment:
Does this need to be a separate section? Why does the user need to know that it's specifically for STAC?